README.md

```diff
@@ -67,6 +67,48 @@ Complete the following steps before you deploy the Secure Landing Zone module.
 
     Select **All Identity and Access enabled services** when you assign access to the group.
 
+### Steps to prevent Destroy and recreation of cluster.
+1. Prior to `terraform plan` and `terraform apply`. Edit `moved_config.tf`
+2. Add the following lines to the terraform file.
+  ```
+  moved {
+    from = ibm_container_vpc_cluster.cluster["<workload_cluster_name>"]
+    to   = module.cluster["<workload_cluster_name>"].ibm_container_vpc_cluster.cluster[0]
+  }
+  moved {
+    from = ibm_container_vpc_cluster.cluster["<management_cluster_name>"]
+    to   = module.cluster["<management_cluster_name>"].ibm_container_vpc_cluster.cluster[0]
+  }
+  ```
+3. Make sure to replace the `<workload_cluster_name>` with the actual workload cluster name. Similarly `<management_cluster_name>` with the management cluster name.
+4. Go ahead with `terraform plan` and `terraform apply`.
+
+#### Alternative approach using terraform cli:
```

Comment:
Because there are alternatives, I suggest you list them after the opening description. Something like, 

Choose the method that matches how you work with the module: 
- Modify the Terraform code
- Use the Terraform CLI
- Use Schematics

Modify this so that it makes sense though.